### DISCUSSION TITLE
Any information could be public for phase 2 and 3 test data

### DISCUSSION THREAD

**Discussion by hongyingyue** [PaddlePaddle/PaddleSpatial#220] [BaiduKDDCup2022]
Like we know the phase 1 has 195 test samples, and they are after the training 245 days.
Any information about current test data and final private test data?

Thanks

**Comment by AlvinAi96** [comment]
My confusions are : 
1. How many test samples do the 2nd and 3rd phases have respectively? 
2. Does you split the current test samples into two proportions? one of them is evaluated and shown in the 2nd LB, and another one is also evaluated right now but is hidden for the 3rd phase?

**Reply by hongyingyue** [reply]
It seems the 2nd phase has 219 samples according to the latest update, but other info is still unknown

**Reply by ramber1836** [reply]
> My confusions are :
> 
> 2. Does you split the current test samples into two proportions? one of them is evaluated and shown in the 2nd LB, and another one is also evaluated right now but is hidden for the 3rd phase?

No.
